stable/k8s-spot-termination-handler/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.13.7-1"
 description: The K8s Spot Termination handler handles draining AWS Spot Instances in response to termination requests.
 name: k8s-spot-termination-handler
-version: 1.4.3
+version: 1.4.4
```

Comment:
Looks like you need to bump it
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/helm_charts/16946/pull-charts-e2e/1173406394401951744#0:build-log.txt%3A181

Reply:
Looks like the Chart version changed after my PR. Should be fixed now